### PR TITLE
DictPostFixes

### PR DESCRIPTION
**Pull request by rijobro**
Follows from https://github.com/Project-MONAI/MONAI/pull/3605.

### Description
Replace `"meta_dict"` with `DictPostFixes.META`.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).

### CODE REVIEW

**Comment by ericspod** [comment]
We may want to commit #3605 first to prevent conflicts if git gets confused about two changes in very close proximity.

File: monai/utils/enums.py [hunk 1]
```diff
@@ -259,6 +260,13 @@ class CommonKeys:
     LOSS = "loss"
 
 
+class DictPostFixes:
+    """Post-fixes."""
+
+    META = "meta_dict"
+    ORIG_META = "orig_meta_dict"
```
Comment (wyli) [inline monai/utils/enums.py:267]: shall we add this refactoring:
```py
class DictPostFixes:
    """Post-fixes."""

    META = "meta_dict"
    ORIG_META = "orig_meta_dict"

    @staticmethod
    def meta_key(key):
        return f"{key}_{DictPostFixes.META}"
```
usage:
```py
DictPostFixes.meta_key("image")  # "image_meata_dict"
```

Comment (rijobro) [reply monai/utils/enums.py:267]: Great idea, I'll update now.

**Review by wyli — APPROVED** [review]
thanks, this is the first step towards https://github.com/Project-MONAI/MONAI/milestone/31. cc@Nic-Ma please let me know any concerns otherwise I'll merge it today...

**Review by Nic-Ma — APPROVED** [review]
Looks good to me.

Thanks for the enhancement.

**Pull request merged**